### PR TITLE
PMM 10598 manage dashboards

### PR DESCRIPTION
**Pull request by rasika-chivate**

### CODE REVIEW

File: docs/details/dashboards/dashboard-manage-dashboards.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+To delete multiple dashboards at once:
+
+From the side menu, go to <i class="uil uil-apps"></i> *Dashboards > Browse* and check the dashboards that you want to delete, and click *Delete*.
+
+
+
+
+![!image](../../_images/PMM_Delete_multiple_dashbaords.png)
+
+### Move dashboards from one folder to another
```
Comment (dygksy) [inline docs/details/dashboards/dashboard-manage-dashboards.md:42]: There are 2 ways. One of them is already written by you. Other way is here: https://www.percona.com/blog/pmm-v2-30-new-dashboards-improved-dbaas-experience-free-k8s-cluster-for-testing-and-more/

File: docs/details/dashboards/dashboard-manage-dashboards.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+6. In the Home Dashboard field, select the dashboard that you want to use for your home dashboard.
+7. Click *Save*.
+
+
+### Set your Personal Home Dashboard
+
+1. From the main menu, go to <i class="uil uil-apps"></i> *Dashboards > Browse* and select the dashboard you want to set as your home dashboard.
+2. Click the <i class="uil uil-star"></i> star next to the dashboard title to mark it as a favorite.
+
+    ![!image](../../_images/PMM_click_to_add_favorite_dashbaord.png)
```
Comment (denisok) [inline docs/details/dashboards/dashboard-manage-dashboards.md:107]: this image has `N/A` , maybe better to get similar image with some pretty image.

File: docs/details/dashboards/dashboard-manage-dashboards.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+This section describes how to delete multiple dashboards, move dashboards from one folder to another and navigate to a folder page where you can assign folder and dashboard permissions.
+
+### Delete multiple dashboards
+
+To delete multiple dashboards at once:
+
+From the side menu, go to <i class="uil uil-apps"></i> *Dashboards > Browse* and check the dashboards that you want to delete, and click *Delete*.
+
+
+![!image](../../_images/PMM_Delete_multiple_dashbaords.png)
```
[the remark this answers is not in the document]
Comment (rasika-chivate) [reply docs/details/dashboards/dashboard-manage-dashboards.md:38]: Fixed

File: docs/details/dashboards/dashboard-manage-dashboards.md (hunk 2 above)
Comment (rasika-chivate) [reply docs/details/dashboards/dashboard-manage-dashboards.md]: Fixed

**Review by anton-bystrov — COMMENTED** [review]
Typo in screenshots filenames "dashbaord".

File: docs/details/dashboards/dashboard-manage-dashboards.md (hunk 1 above)
Comment (rasika-chivate) [reply docs/details/dashboards/dashboard-manage-dashboards.md:40]: Done

**Comment by rasika-chivate** [comment]
> @rasika-chivate I think a new screenshot of docs/_images/PMM_click_to_add_favorite_dashbaord.png with data would seem better. On this ss, there is no data and it looks like something wrong :(

**Comment by rasika-chivate** [comment]
@dygksy Updated the SS.

**Comment by rasika-chivate** [comment]
> 

Fixed

**Review by dygksy — APPROVED** [review]

**Review by matejkubinec — APPROVED** [review]

**Pull request merged**